.travis.yml

```diff
@@ -1,19 +1,18 @@
 rvm:
   - 2.1.10
-  - 2.2.5
-  - 2.3.1
-  - 2.4.0
+  - 2.2.10
+  - 2.3.8
+  - 2.4.5
+  - 2.5.3
   - jruby-19mode
-  - jruby-9.0.5.0
+  - jruby-9.1.8.0
```

Comment:
Why do you specify 9.1.8, sorry I'm not familiar with jruby versioning?

Reply:
It's just latest one

Reply:
Yes latest one is 9.2.4.0 .